docs/reference/cinnamon-tutorials/write-applet.xml

```diff
@@ -90,28 +95,36 @@
       <programlisting>
         const Applet = imports.ui.applet;
         const Util = imports.misc.util;
+        const GLib = imports.gi.GLib;
+
+        /**
+         * Variables (and other constants)
+         */
+        var user = GLib.getenv("USER"); // acts like the shell command: echo -n $USER
+
+        user = user.charAt(0).toUpperCase() + user.slice(1); // Capitalize 'user'
+
+        /**
+         * Declaration of the 'NotifyMe' class, which extends
+         * the 'IconApplet' class of the 'Applet' library.
+         */
+        class NotifyMe extends Applet.IconApplet {
+            constructor (metadata, orientation, panelHeight, instance_id) {
+                super(orientation, panelHeight, instance_id);
+                this.set_applet_icon_symbolic_name("bell-notif");
+                this.set_applet_tooltip("Click here to send a notification");
+            }
 
-        function MyApplet(orientation, panel_height, instance_id) {
-            this._init(orientation, panel_height, instance_id);
-        }
-
-        MyApplet.prototype = {
-            __proto__: Applet.IconApplet.prototype,
-
-            _init: function(orientation, panel_height, instance_id) {
-                Applet.IconApplet.prototype._init.call(this, orientation, panel_height, instance_id);
-
-                this.set_applet_icon_name("force-exit");
-                this.set_applet_tooltip(_("Click here to kill a window"));
-            },
-
-            on_applet_clicked: function() {
-                Util.spawn(['xkill']);
+            on_applet_clicked(event) {
+                // Please consult 'man notify-send'.
+                let command = `bash -c 'notify-send -u normal `;
+                command += `"Thank you ${user}" "for improving Cinnamon!"'`;
+                Util.spawnCommandLineAsync(command)
```

Comment:
Maybe a comment emphasizing that async should always try to be used, and why.

Reply:
Done!